packages/in_app_purchase/lib/src/in_app_purchase/purchase_details.dart

```diff
@@ -107,6 +110,33 @@ class PurchaseParam {
 
   /// The 'sandboxTesting' is only available on iOS, set it to `true` for testing in AppStore's sandbox environment. The default value is `false`.
   final bool sandboxTesting;
+
+  /// The 'changeSubscriptionParam' is only available on Android, for upgrading or
+  /// downgrading an existing subscription.
```

Comment:
Let's also add some documentation to explain how exactly changing subscription plan can be done on iOS.

Reply:
Done